torchrl/data/tensordict/tensordict.py

```diff
@@ -4732,3 +4732,37 @@ def _expand_to_match_shape(parent_batch_size, tensor, self_batch_dims, self_devi
 #
 #
 # _register_pytree_node(TensorDict, _flatten_tensordict, _unflatten_tensordict)
+
+
+def make_tensordict(
+    source: Union[TensorDictBase, dict],
```

Comment:
do you think we could make this
`make_tensordict(a=..., b=..., )` instead than a dict?